engine/rpcserver.go

```diff
@@ -861,15 +871,15 @@ func (s *RPCServer) GetOrder(_ context.Context, r *gctrpc.GetOrderRequest) (*gct
 		AssetType:     result.AssetType.String(),
 		OrderSide:     result.Side.String(),
 		OrderType:     result.Type.String(),
-		CreationTime:  result.Date.Unix(),
+		CreationTime:  creationTime,
```

Comment:
Is there a specific reason you have done the checks above this? I would just revert back to result.Date.Unix() and closetime.